examples/todolist-react/src/containers/TodoItem.js

```diff
@@ -5,6 +5,7 @@ import { withActuator } from "state-actuator/lib/react";
 
 import useDoubleClick from "../hooks/useDoubleClick";
 import useOnEnter from "../hooks/useOnEnter";
+import { SEND_TO_PARENT } from "../utils";
```

Comment:
I might add this to state-actuator itself, but this is a good idea for sure.

Reply:
Good call, I will remove it!